roles/artemis/templates/application-prod.yml.j2

```
@@ -355,7 +379,11 @@ info:
 {% endif %}
   textAssessmentAnalyticsEnabled: true
   sentry:
-    dsn: https://e52d0b9b6b61769f50b088634b4bc781@sentry.ase.in.tum.de/2
+{% if sentry.dsn is defined and sentry.dsn != '' %}
+    dsn: ${sentry.dsn}
```

```suggestion
    dsn: {{ sentry.dsn }}
```

Should not really matter, but you are refering to the ansible variable `sentry` in the if statement and I think this should be consistent.